src/test/java/com/example/pfairplayservice/jpa/repository/MemberRepositoryIntegrationTest.java

```diff
@@ -15,42 +20,76 @@
 import org.springframework.test.context.jdbc.Sql;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 
-import java.util.Date;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
 import java.util.Optional;
 
 @ExtendWith(SpringExtension.class)
 @DataJpaTest
 public class MemberRepositoryIntegrationTest {
 
     @Autowired
-    private TestEntityManager entityManager;
+    private MemberRepository memberRepository;
 
     @Autowired
-    private MemberRepository memberRepository;
+    private TeamRepository teamRepository;
+
+    @Autowired
+    private MemberTeamRepository memberTeamRepository;
 
     @Test
     public void whenFindById_thenReturnMemberEntity() {
-        MemberEntity memberEntity = MemberEntity
-                .builder()
-                .id("ryoo")
-                .password("testpw")
-                .address("testad")
-                .birthday(new Date())
-                .joinDate(new Date())
-                .level(1)
-                .preferPosition(1)
-                .phoneNumber("010")
-                .recentLoginDate(new Date())
-                .phoneNumberDisclosureOption(1)
-                .name("ryoo2")
-                .build();
-
-        entityManager.persist(memberEntity);
-        entityManager.flush();
 
+        // given
+        MemberEntity memberEntity = TestEntityGenerator.generateMemberEntity();
+
+        memberRepository.save(memberEntity);
+        memberRepository.flush();
+
+        // when
         Optional<MemberEntity> found = memberRepository.findById(memberEntity.getUid());
 
+        // then
         assertThat(found.get().getUid(), is(equalTo(memberEntity.getUid())));
     }
 
+    @Test
+    public void whenFindByMemberTeamTid_thenReturnMemberEntityList() {
+
+        // given
+        MemberEntity memberEntity0 = TestEntityGenerator.generateMemberEntity();
+        MemberEntity memberEntity1 = TestEntityGenerator.generateMemberEntity();
+        MemberEntity memberEntity2 = TestEntityGenerator.generateMemberEntity();
+
+        TeamEntity teamEntity = TestEntityGenerator.generateTeamEntity(memberEntity0);
+
+        memberRepository.save(memberEntity0);
+        memberRepository.save(memberEntity1);
+        memberRepository.save(memberEntity2);
+        teamRepository.save(teamEntity);
```

Comment:
Remove all records which are saved in the given area after the `then` section (L92).
Because it can affect another test case.